modules/islandora_google_scholar/islandora_google_scholar.module

```diff
@@ -66,15 +69,16 @@ function islandora_google_scholar_create_meta_tags($object) {
               }
             }
           }
-        }
         if (isset($name_parts['last_name']) && isset($name_parts['first_name'])) {
           $tags['citation_author'][] = $name_parts['last_name'] . ', ' . $name_parts['first_name'];
         }
         elseif (isset($name_parts['no_type'])) {
           $tags['citation_author'][] = $name_parts['no_type'];
         }
+		elseif (isset($name_displayForm)) {
```

Comment:
So. (not requesting change, just a question) if no_type is present, whatever the value, then displayForm is not taken in account? So what happens if the values are empty but the xpath resolves?

Reply:
Maybe it is better to let displayForm take precedent?

Reply:
I really don't know what is best in terms of precedence for scholar uses (it seems to me that if displayForm is there it would not match the last name, first name format that the other imposes but on the other side it feels if its there its the intended use too). What i'm sure of is that in any case you need to make sure there is an actual value and not just the element there using xpath. Islandora does have a clean up XSLT present that gets rid of empties, but that behavior is not enforced and required everywhere. Also, `mods:displayForm` is unbound in terms of max values, means you could have more than one inside a single name and`::xpath` returns always an array of Simple XML elements or FALSE. So you need to iterate over the results (even if a single one is expected). So  `$name_displayForm = $name_xml->xpath('mods:displayForm') == array` of displayForm elements, and then the (string) hack there converts that object via https://www.php.net/manual/en/simplexmlelement.tostring.php into the inner value.

Reply:
I have added (string) to $name_displayform. I will be checking if $name_displayForm is empty. I will let displayForm take precedent over no_part